tests/test_mailchimp.py

```diff
@@ -18,69 +20,57 @@ def setUp(self, *args, **kwargs):
         with self.context:
             handlers.celery_before_request()
 
-    @mock.patch('website.mailchimp_utils.get_mailchimp_api')
-    def test_get_list_id_from_name(self, mock_get_mailchimp_api):
```

Comment:
Removed as `get_list_id_from_from_name` no longer makes external calls -- it's a key lookup in `MAILCHIMP_LIST_MAP`.